### PR TITLE
Add TextCaseConverter

### PR DESCRIPTION
**Pull request by bijington**
Documentation added for .NET MAUI Community Toolkit proposals:

- `[Proposal] TextCaseConverter` <https://github.com/CommunityToolkit/Maui/issues/76>

### CODE REVIEW

**Comment by PRMerger20** [comment]
## PRMerger Results
 | Issue | Description |
 --- | --- |
 Added File(s) | This PR contains added files. New files require human review.
 Yaml File(s) | This PR includes changes to .yml file(s) owned by another author.
 File Change Percent | This PR contains file(s) with more than 20% file change.

**Comment by PRMerger3** [comment]
## PRMerger Results
 | Issue | Description |
 --- | --- |
 Added File(s) | This PR contains added files. New files require human review.
 Yaml File(s) | This PR includes changes to .yml file(s) owned by another author.
 File Change Percent | This PR contains file(s) with more than 20% file change.

**Comment by PRMerger4** [comment]
## PRMerger Results
 | Issue | Description |
 --- | --- |
 Added File(s) | This PR contains added files. New files require human review.
 Yaml File(s) | This PR includes changes to .yml file(s) owned by another author.
 File Change Percent | This PR contains file(s) with more than 20% file change.

**Comment by PRMerger10** [comment]
## PRMerger Results
 | Issue | Description |
 --- | --- |
 Added File(s) | This PR contains added files. New files require human review.
 Yaml File(s) | This PR includes changes to .yml file(s) owned by another author.
 File Change Percent | This PR contains file(s) with more than 20% file change.

**Comment by PRMerger19** [comment]
## PRMerger Results
 | Issue | Description |
 --- | --- |
 Added File(s) | This PR contains added files. New files require human review.
 Yaml File(s) | This PR includes changes to .yml file(s) owned by another author.
 File Change Percent | This PR contains file(s) with more than 20% file change.

File: docs/maui/converters/text-case-converter.md [hunk 1]
```diff
@@ -0,0 +1,109 @@
...
+```csharp
+using CommunityToolkit.Maui.Markup;
+
+class TextCaseConverterPage : ContentPage
+{
+    public TextCaseConverterPage()
+    {
+        Content = new Label()
+            .Bind(
+                Label.Text,
```
Comment (jfversluis) [inline docs/maui/converters/text-case-converter.md:81]:
```suggestion
                Label.TextProperty,
```

**Comment by PRMerger18** [comment]
## PRMerger Results
 | Issue | Description |
 --- | --- |
 Added File(s) | This PR contains added files. New files require human review.
 Yaml File(s) | This PR includes changes to .yml file(s) owned by another author.
 File Change Percent | This PR contains file(s) with more than 20% file change.

**Comment by PRMerger18** [comment]
## PRMerger Results
 | Issue | Description |
 --- | --- |
 Added File(s) | This PR contains added files. New files require human review.
 Yaml File(s) | This PR includes changes to .yml file(s) owned by another author.
 File Change Percent | This PR contains file(s) with more than 20% file change.

File: docs/maui/converters/text-case-converter.md (hunk 1 above)
Comment (bijington) [reply docs/maui/converters/text-case-converter.md]: Great spot thanks!

**Comment by PRMerger10** [comment]
## PRMerger Results
 | Issue | Description |
 --- | --- |
 Added File(s) | This PR contains added files. New files require human review.
 Yaml File(s) | This PR includes changes to .yml file(s) owned by another author.
 File Change Percent | This PR contains file(s) with more than 20% file change.

**Review by jfversluis — APPROVED** [review]

**Pull request merged**